owncloudApp/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.kt

```diff
@@ -1647,6 +1657,36 @@ class FileDisplayActivity : FileActivity(), FileFragment.ContainerActivity, OnEn
         manageOptionLockSelected(type)
     }
 
+    private fun manageDataIntent(uri: Uri?) {
+        if (uri != null && AccountUtils.getAccounts(applicationContext).isEmpty()) {
+            showMessageInToast(getString(R.string.no_account_configured))
+        } else if (uri != null && AccountUtils.getAccounts(applicationContext).size == 1) {
+            isFileDiscovered(uri).let { OCFile ->
+                if (OCFile != null) {
+                    manageItem(OCFile)
+                } else {
+                    showMessageInToast(getString(R.string.no_file_found))
+                }
+            }
+        }
+    }
+
+    private fun isFileDiscovered(uri: Uri?): OCFile? = storageManager.getFileByPrivateLink(uri.toString())
```

Comment:
`isSomethingSomehow()` functions are expected to return booleans. If we are returning a specific object, I would use a different name.

Reply:
You're right! Fixed